docker-compose-osx.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.6"
+services:
+  node:
+    build: .
+    user: "node"
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=development
+    volumes:
+      - codebase-docker-sync:/home/node/app
+      - codebase-docker-sync:/home/node/app/book
+    ports:
+      - "5000:5000"
+
+volumes:
```

Comment:
out of curiosity, why is there an osx-specific docker compose file?

Reply:
Hey man, it's used to fix the sync issues for small files on OSX.
https://docker-sync.readthedocs.io/en/latest/